x/modelinfo/client/rest/tx.go

```diff
@@ -23,35 +23,14 @@ import (
 	"github.com/zigbee-alliance/distributed-compliance-ledger/x/modelinfo/internal/types"
 )
 
-//nolint:maligned
 type AddModelInfoRequest struct {
-	BaseReq                  restTypes.BaseReq `json:"base_req"`
-	VID                      uint16            `json:"vid"`
-	PID                      uint16            `json:"pid"`
-	CID                      uint16            `json:"cid,omitempty"`
-	Version                  string            `json:"version,omitempty"`
-	Name                     string            `json:"name"`
-	Description              string            `json:"description"`
-	SKU                      string            `json:"sku"`
-	HardwareVersion          string            `json:"hardware_version"`
-	FirmwareVersion          string            `json:"firmware_version"`
-	OtaURL                   string            `json:"ota_url,omitempty"`
-	OtaChecksum              string            `json:"ota_checksum,omitempty"`
-	OtaChecksumType          string            `json:"ota_checksum_type,omitempty"`
-	Custom                   string            `json:"custom,omitempty"`
-	TisOrTrpTestingCompleted bool              `json:"tis_or_trp_testing_completed"`
+	Model   types.Model       `json:"model"`
+	BaseReq restTypes.BaseReq `json:"base_req"`
 }
 
-//nolint:maligned
 type UpdateModelInfoRequest struct {
-	BaseReq                  restTypes.BaseReq `json:"base_req"`
-	VID                      uint16            `json:"vid"`
-	PID                      uint16            `json:"pid"`
-	CID                      uint16            `json:"cid,omitempty"`
-	Description              string            `json:"description,omitempty"`
-	OtaURL                   string            `json:"ota_url,omitempty"`
-	Custom                   string            `json:"custom,omitempty"`
-	TisOrTrpTestingCompleted bool              `json:"tis_or_trp_testing_completed"`
+	Model   types.Model       `json:"model"`
```

Comment:
I believe we don't allow changing of any field in Edit command. So, should we use another type here, not `types.Model` which has all the model fields?